samples/emoji-search/presenter/src/commonMain/kotlin/com/example/redwood/emojisearch/presenter/EmojiSearch.kt

```diff
@@ -60,11 +62,25 @@ interface Navigator {
   fun openUrl(url: String)
 }
 
-@OptIn(ExperimentalRedwoodLazyLayoutApi::class)
 @Composable
 fun EmojiSearch(
   httpClient: HttpClient,
   navigator: Navigator,
+) {
+  // Different variants of EmojiSearch:
+
+  // Flex Containers
+  NestedFlexBoxContainers(httpClient, navigator)
+
+  // Lazy Column
+  // LazyColumn(httpClient, navigator)
```

Comment:
I'm a bit wary of making this sample too complicated, as it's our only real Treehouse one. For now we can leave this as-is, but it's probably wise to add a playground-like app in the future were we can capture more esoteric experiments.

Reply:
Also we plan to have automated UI tests against this app to validate behavior as an integration test for each platform (#537), which is another reason we shouldn't add a ton of features to it.

Reply:
Is the EmojiSample mean to only develop/validate/test the LazyColumn? I think we should start adding more complex examples somewhere in the redwood sample code since that's what we'll end up needing downstream from redwood and running into redwood issues at that stage really bogs down the iteration time.

Since we're using EmojiSample as the main development environment in redwood, it should stress test more complex (real world) cases to get ahead of regressions and not require feedback signals from treehouse guest app development

Reply:
It's the only sample that demonstrates Treehouse usage. The counter sample demonstrates raw Redwood.

This is why I would prefer both be kept as simple as possible. Each is there to demonstrate the structure of the technology (which is already complex) and so a complex application can distract or mask what's going on with the setup.

So I would prefer if long-term we either created a playground-style sample in which we can put a bunch of screens with more targeted demonstrations of particular components, or we built a separate sample which is just a more complex app. Or we could do both!

The repo search sample is one which is solely meant to demo LazyColumn usage, I believe. It would fit well within a playground app, or we could expand it to be an exceedingly basic GitHub browser app as a more complex example.

We don't have to do anything in this PR right now, though.

Reply:
Got it. I'll merge this for now and we can spin up a playground sample (or one dedicated to flexbox).

Let's add a README to https://github.com/cashapp/redwood/tree/trunk/samples to make it clear what each sample is intended to demonstrate